scripts/publish-pypi

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
You might want to pick up the script from `utils` repo for `scripts/publish`. It does more error-checking and is generic as to what repo uses it so can probably be used directly.